server/models/appointment.ts

```diff
@@ -2,11 +2,21 @@ import Address from './address'
 import { AppointmentDeliveryType } from './appointmentDeliveryType'
 import SessionFeedback from './sessionFeedback'
 
-export default interface Appointment {
+export interface InitialAssessmentAppointment extends Appointment {
```

Comment:
Changes to this file are the bulk of this commit - the rest is really updating the rest of the codebase to use these interfaces.